### PR TITLE
Update info and changelog for WDMAM 2.1

### PR DESCRIPTION
**Pull request by Esteban82**
I update the info for WDMAM 2.1 and the changelog. 

In the page I put the version in the title. 

I add WIP until the remote data sets are updated.

### CODE REVIEW

**Comment by seisman** [comment]
> In the page I put the version in the title.

I don't like the version in the title. It gives me the impression that I can access old versions of a dataset, which is not true.

File: docs/changes.rst [hunk 1]
```diff
@@ -1,6 +1,11 @@
 Changelog
 ---------
 
+2023-08-11
+^^^^^^^^^^
+
+- Update ``earth_wdmam`` source from version 2.0 to 2.1 [`Choi et al., 2023`_].
```
Comment (seisman) [inline docs/changes.rst:7]: Maybe we can have a more consistent changelog style like:

- ``dataset_name``: Summary of the change [the reference or the issue if any]

```suggestion
- ``earth_wdmam``: Update dataset source from version 2.0 to 2.1 [`Choi et al., 2023`_].
```

**Comment by Esteban82** [comment]
> I don't like the version in the title. It gives me the impression that I can access old versions of a dataset, which is not true.

Ok. My only intention is to see easily see the version. Maybe we can put it somewhere else that is also visible. Any suggestion?

**Comment by Esteban82** [comment]
I remove the version from the title. It can be discusse on another issue.

**Comment by PaulWessel** [comment]
Maybe on the page describing the data sets we can state that we only offer the latest version of a data set.  If someone needs an oder version for comparisons etc they will have to get that separately.

**Comment by Esteban82** [comment]
I think this is ready to merge.

**Review by PaulWessel — APPROVED** [review]

**Review by seisman — APPROVED** [review]

**Comment by PaulWessel** [comment]
Which web page?

**Comment by Esteban82** [comment]
https://www.generic-mapping-tools.org/remote-datasets/earth-wdmam.html

**Pull request merged**